src/en/help-azure.md

```diff
@@ -20,151 +21,76 @@ running:
-following command:
+!!! Note:
+    Credentials on the Azure cloud have been reported to expire. If a
+    previously working setup suddenly behaves as if incorrect credentials are
+    being used then you may need to update the credentials on the controller.
+    See [Updating remote credentials][updating-remote-credentials] for
+    guidance.
 
-```bash
-az login
-```
-The above command will prompt you to open a browser with a specific URL and
-enter a provided authentication code.
+### Install the CLI tool
 
-After entering the code and pressing continue, you will be asked to select the
-Microsoft account you'd like associated with the Azure CLI. 
+You will need to import your Azure credentials into Juju using the
+[Azure CLI 2.0][azurecli] tool from Microsoft.
 
-Back on the command line, the output from `az login` will have concluded by
-displaying the credentials for your account: 
-
-```yaml
-[
-  {
-    "cloudName": "AzureCloud",
-    "id": "f717c8c1-8e5e-4d38-be7f-ed1e1c879e18",
-    "isDefault": true,
-    "name": "Pay-As-You-Go",
-    "state": "Enabled",
-    "tenantId": "0fb95fd9-f42f-4c78-94c9-e3d01c2bc5af",
-    "user": {
-      "name": "javierlarin72@gmail.com",
-      "type": "user"
-    }
-  }
-]
-```
-
-Your Azure credentials can now be added to Juju by running the command:
+Ubuntu/Linux users can install Azure CLI 2.0 with the following command:
 
 ```bash
-juju add-credential azure
-```
-
-You will first be asked for an arbitrary credential name, which you choose
-for yourself.  This will be how you remember and refer to this Azure credential
-in Juju. The second question will ask you to select an 'Auth Type' from the
-following two options:
-
-- interactive
-- service-principal-secret
-
-The default option is `interactive` and you can either type 'interactive' or
-press 'Enter' to continue. 
-
-The 'interactive' option is far quicker and easier than manually adding
-credentials via the 'service-principal-secret' option, but instructions for
-this are covered in the [Manually adding
-credentials](#manually-adding-credentials) section below. Follow this manual
-process if the 'interactive' option fails, or you want to configure automated
-testing in a new environment.
-
-You then will be asked for your Azure subscription id (***id*** from the
-`az login` credentials output, as shown above). Entering this is optional, as your Azure
-credentials will be automatically retrieved by Juju by pressing enter. 
-
-Once the authentication is successful, you will see the following:
-
-```no-highlight
-Credentials added for cloud azure.
+curl -L https://aka.ms/InstallAzureCli | bash
 ```
 
-If you want to check that the credentials were successfully added, use the
-`juju credentials` command. You will see your Azure credentials listed.
+!!! Note:
+    For instructions that cover installing Azure CLI on Microsoft Windows and
+    Apple macOS, see Microsoft's [Install Azure CLI 2.0][azuretwoinstall]
```

Comment:
macOS?

Reply:
Should it be something else? I've never used Apple products.

Reply:
damn mine eyes. I was sure this was lowercase when I commented! Sorry. And yeah, macOS is the current Apple preference I think.

Reply:
:+1: